fix: changes to avoid sending logs to logdna

imprateeksh opened the pull request:
### Description

This is regarding the issue where we have to discontinue the usage of LogDNA and use IBM Cloud Logs. While the other solution is in progress but this change is to stop sending logs to logdna. 

### Release required?
<!--- Identify the type of release. For information about the changes in a semantic versioning release, see [Release versioning](https://terraform-ibm-modules.github.io/documentation/#/versioning). --->

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

<!--- If a release is required, replace this text with information that users need to know about the release. Write the release notes to help users understand the changes, and include information about how to update from the previous version.

Your notes help the merger write the commit message for the PR that is published in the release notes for the module. --->

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

ocofaigh commented on module-assets/ci/run-tests.sh:
```diff
@@ -147,61 +147,10 @@ if [ ${IS_PR} == true ]; then
-    fi
+    # Assign tags
+    export ICL_TAGS="${REPO_NAME}-PR${PR_NUM},commit-${COMMIT_ID},pipeline-${PIPELINE_ID},env-${ENV_TAG}"
+    echo "$ICL_TAGS" # Keeping this for now as this information will be required when using ICL
```
Not sure we will use ICL here, but instead maybe leverage github checks. Remove this for now

imprateeksh replied:
sure done

ocofaigh reviewed (changes requested):
see comment

imprateeksh commented:
Removed ICL reference and also removed some unused variables.

<img width="690" alt="image" src="https://github.com/user-attachments/assets/256e3499-09c5-43b4-acf6-817542f177a0">
